website/project/views/node.py

```diff
@@ -49,7 +50,8 @@
 def edit_node(auth, node, **kwargs):
     post_data = request.json
     edited_field = post_data.get('name')
-    value = strip_html(post_data.get('value', ''))
+    value = post_data.get('value', '')
```

Comment:
With the changes we made back and forth, I think the old version of this function makes more sense ( https://github.com/CenterForOpenScience/osf.io/pull/4230/files#diff-033d4b3df3102b7d1aace074938585e9L52 ), and the changes made here can be reverted.

With that done, we can take `strip_html` back out of `set_title` as well. (https://github.com/CenterForOpenScience/osf.io/pull/4230/files#diff-b4b50a4acab364147a53259fa3fe9b3fR1573 )

Going back to the old version for the two places linked will ensure that `set_title` and `set_description` work the same way.